moped-editor/src/views/projects/projectView/ProjectView.js

```diff
@@ -411,6 +458,24 @@ const ProjectView = () => {
                           </Breadcrumbs>
                         </Box>
                       </Grid>
+                      <Grid item xs={1} md={1}>
+                        <Box
+                          className={classes.followDiv}
+                          onClick={() => handleFollowProject()}
+                        >
+                          {isFollowing ? (
+                            <Tooltip title="Unfollow">
+                              <BookmarkIcon className={classes.unfollowIcon} />
+                            </Tooltip>
+                          ) : (
+                            <Tooltip title="Follow">
+                              <BookmarkBorderIcon
+                                className={classes.followIcon}
+                              />
+                            </Tooltip>
+                          )}
```

Comment:
One way to make this code more concise would be to only put a ternary operator on the differences in the props instead of the whole nested components. Something like the following:

```js
<Tooltip title={isFollowing ? "Unfollow" : "Follow" }>
  <BookmarkIcon className={isFollowing ? classes.unfollowIcon : classes.followIcon} />
</Tooltip>
```

Reply:
I think you could replace lines 466-476 with just those 3 lines.

Reply:
This is so much better, thank you for the suggestion!

Reply:
The icon type needs to be conditionally rendered as well, so the final code looks like this:

`                          <Tooltip title={isFollowing ? "Unfollow" : "Follow"}>
                            {isFollowing ? (
                              <BookmarkIcon className={classes.unfollowIcon} />
                            ) : (
                              <BookmarkBorderIcon
                                className={classes.followIcon}
                              />
                            )}
                          </Tooltip>`